vars/onMaster.groovy

```diff
@@ -18,9 +18,12 @@ def call(timeoutString, Closure body) {
                 // To export BOTO_CONFIG, for some reason, worker did not
                 // source the .profile or .bashrc anymore.
                 withEnv(["BOTO_CONFIG=${env.HOME}/.boto",
-                         "PATH=/usr/local/google_appengine:" +
+                         "PATH=" +
+                         "/home/ubuntu/webapp-workspace/devtools/khan-linter/bin:" +
+                         "/var/lib/jenkins/repositories/khan-linter/bin" +
+                         "/usr/local/google_appengine:" +
                          "/home/ubuntu/google-cloud-sdk/bin:" +
-                         "${env.HOME}/git-bigfile/bin:" +
```

Comment:
I'm in agreement that git-bigfile _shouldn't_ be in use, but I wonder if it might be safer to leave this for now and remove it in a follow up commit so it can be independently reverted if it's a problem?

Reply:
I'm comfortable removing git-bigfile, nothing in this repo mentions the word 'bigfile' anymore after this.

/usr/local/bin is still there -- it's part of the default `$PATH` (which is included here in `env.PATH` on the last line).